index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles2.css">
+    <title>Document</title>
+</head>
+
+
+<body>
+
+<!--header / top part-->>
+    
+  <nav class="topbar">
+    <h1>Welcome</h1>
```

Comment:
- [ ] Since we are considering this as your logo, wrap it around `a` tags.

Reply:
Ok, thanks for the advice. Done.